dlt/cli/pipeline_command.py

```diff
@@ -101,13 +114,23 @@ def _display_pending_packages() -> Tuple[Sequence[str], Sequence[str]]:
 
     if operation == "show":
         from dlt.common.runtime import signals
-        from dlt.helpers import streamlit_helper
+        from dlt.helpers.streamlit_app import index
+
+        with signals.delayed_signals(), custom_environ({"DLT_PIPELINES_DIR": pipelines_dir or ""}):
```

Comment:
OK so if we use custom env: why not to pass this as another argument? and use a env in the tests? it is way more clean

Reply:
Done